sdk/cosmos/azure-cosmos-kafka-connect/src/main/java/com/azure/cosmos/kafka/connect/implementation/sink/CosmosPointWriter.java

```diff
@@ -146,6 +153,41 @@ private void deleteWithRetry(CosmosAsyncContainer container, SinkOperation sinkO
         );
     }
 
+    private void patchWithRetry(CosmosAsyncContainer container, SinkOperation sinkOperation) {
+        executeWithRetry(
+            (operation) -> {
+                CosmosPatchItemRequestOptions patchItemRequestOptions = new CosmosPatchItemRequestOptions();
+                CosmosThroughputControlHelper.tryPopulateThroughputControlGroupName(patchItemRequestOptions, this.throughputControlConfig);
+                if (StringUtils.isNotEmpty(this.writeConfig.getCosmosPatchConfig().getFilter())) {
+                    patchItemRequestOptions.setFilterPredicate(this.writeConfig.getCosmosPatchConfig().getFilter());
+                }
+
+                return ImplementationBridgeHelpers
```

Comment:
I am wondering whether we really need ItemPatch - (given the limit on 10 properties) - but I assume you plan to add the Update write strategy at a later time?